fava_investor/common/favainvestorapi.py

```diff
@@ -23,7 +24,18 @@ def root_tree(self):
         return self.ledger.root_tree
 
     def query_func(self, sql):
-        contents, rtypes, rrows = self.ledger.query_shell.execute_query(sql)
+        # Based on the fava version, determine if we need to add a new
+        # positional argument to fava's execute_query()
+        split_version = fava_version.split('.')
+        if len(split_version) != 2:
+            split_version = split_version[:2]
+        major, minor = split_version
+        new_execute_query = int(major) > 1 or (int(major) == 1 and int(minor) > 21)
```

Comment:
Would be better to use [packaging.version.parse](https://packaging.pypa.io/en/latest/version.html#packaging.version.parse) for readability, especially given that fava-version specific code may not be one-offs :). Would you mind adding it?

 https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python